docs/reference/modules/snapshots.asciidoc

```diff
@@ -289,6 +289,19 @@ By setting `include_global_state` to false it's possible to prevent the cluster
 the snapshot. By default, the entire snapshot will fail if one or more indices participating in the snapshot don't have
 all primary shards available. This behaviour can be changed by setting `partial` to `true`.
 
+Snapshot could be named according to the date that the snapshot made in a similar way as indices
```

Comment:
How about the following:

Snapshot names can be automatically derived using <<date-math-index-names,date math expressions>>, similarly as when creating new indices. Note that special characters need to be URI encoded. For example, creating a snapshot with the current day in the name, like `snapshot-2018.05.11`, can be achieved with the following command:

...